### ISSUE TITLE
Errors occurred only in the model named Order

### ISSUE THREAD

**Issue by morisobaa** [aws-amplify/amplify-ios#1802]
### Describe the bug

Hi
The following error occurs only when the model name is "Order".
```
type User @model {
    id: ID!
    name: String!
    orders: [Order] @hasMany
}

type Product @model {
    id: ID!
    name: String!
    orders: [Order] @hasMany
}

type Order @model {
    id: ID!
    name: String!
    user: User! @belongsTo
    product: Product! @belongsTo
}
```

```
[logging] near "Order": syntax error in "update Order
set
  "createdAt" = ?,
  "name" = ?,
  "updatedAt" = ?,
  "userOrdersId" = ?,
  "productOrdersId" = ?
where "id" = ?"

DataStoreError: The operation couldn’t be completed. (SQLite.Result error 0.)
Recovery suggestion: The operation couldn’t be completed. (SQLite.Result error 0.)
Caused by:
near "Order": syntax error (code: 1)
```

However, if you change only the model name, it will update successfully.
For example, from "Order" to "Ordering", "Request" or "UserProduct".

- Why did this happen?
- Are there any other words we can't use?

### Steps To Reproduce

```swift
Steps to reproduce the behavior:
1. Define the schema
2. Create and update the Order model
3. See errors
4. Rename the model
5. Create and update a renamed model
6. Confirm that there are no errors
```

### Expected behavior

Docs will explain in advance which words cannot be used.
In [this Docs](https://docs.amplify.aws/cli/graphql/examples-and-solutions/#warehouse-management-system), a model named "Order" is used in the example.

### Amplify Framework Version

8.2.0

### Amplify Categories

DataStore

### Dependency manager

Cocoapods

### Swift version

swift-driver version: 1.45.2 Apple Swift version 5.6

### CLI version

8.2.0

### Xcode version

13.3(13E113)

### Is this a regression?

No

### Device

iPhone12 Simulator

### iOS Version

iOS 15.4

**Comment by atierian** [comment]
Thanks for opening this issue @morisobaa. I'll work on reproducing it and follow up with you here.

**Comment by atierian** [comment]
@morisobaa you mention that you're using Amplify version 8.2.0. That's the CLI version, can you please let us know which version of Amplify iOS you're using?

**Comment by morisobaa** [comment]
How to see the version?

**Comment by thisisabhash** [comment]
Hello, 
If you're consuming Amplify through Cocoapods, you can check the version under the General tab on the pod.
<img width="1150" alt="Screen Shot 2022-05-20 at 4 44 56 PM" src="https://user-images.githubusercontent.com/23294582/169625250-510cd4d7-8177-4ef5-a89a-e867fbe1b5a5.png">

If you're using SPM, you view the package version on the left pane in Xcode.
<img width="577" alt="Screen Shot 2022-05-20 at 4 50 22 PM" src="https://user-images.githubusercontent.com/23294582/169625688-e216c86e-3dfb-4344-a353-0b8f949343a5.png">

**Comment by morisobaa** [comment]
Now it is 1.22.3.

**Comment by thisisabhash** [comment]
Hey @morisobaa, I have merged in a fix for this issue and will update here when we make a release. Thanks.

**Comment by atierian** [comment]
Thanks again for bringing this to our attention @morisobaa!
This fix was including in [Amplify 1.26.0](https://github.com/aws-amplify/amplify-ios/releases/tag/v1.26.0).
